client/src/app/+admin/config/edit-custom-config/edit-custom-config.component.ts

```diff
@@ -29,6 +29,7 @@ export class EditCustomConfigComponent extends FormReactive implements OnInit {
     { value: 50 * 1024 * 1024 * 1024, label: '50GB' }
   ]
   transcodingThreadOptions = [
+    { value: 0, label: 'auto (not optimized)' },
```

Comment:
I don't really understand why not optimized? What value will ffmpeg choose?

Reply:
ffmpeg choose cpu thread count, and this is better to make cpu thread count, but this is better to do cpu thread count -1
Cause on all cpu that isn't high range x86, mid/high range x86_64 or arm v8 if you use all core at 100% your server can become unresponsive durring transcoding.
So all low x86_64, old x86 cpu, arm v6, risc need to keep a core at less than 95% or sometimes all will froze (except transcoding).
(I don't have number but I have a low x86_64 cpu on my laptop so I know, that happend a lot of times, I have to be aware of cpu usage, I have 2c/4t and if all the 4t become fully used my laptop freeze, and it become faster than you think)

But program is niced so this shouldn't appear ?
Yes, but nice require a little bit of cpu time to work but if a process take 100% how it can execute (there is some security for that not happend on last x86, and mid/high x86_64, and arm v8 but not for other (maybe but they didn't work very well (cause this things require a complex cpu))).

Reply:
as for the label, wouldn't a "auto via ffmpeg" or "let ffmpeg choose" be better? This way, users would refer to the ffmpeg documentation.

Reply:
`Auto (via ffmpeg)` :+1: